napari/layers/points/_tests/test_points_mouse_bindings.py

```diff
@@ -841,3 +843,59 @@ def test_drag_start_selection(
         assert False, 'Unreachable code'  # pragma: no cover
     assert layer._drag_box is None
     assert layer._drag_start is None
+
+
+@pytest.mark.xfail(
+    reason='callbacks not properly working when not testing manually.'
+)
+def test_drag_point_with_mouse(create_known_points_layer_2d):
+    layer, n_points, _ = create_known_points_layer_2d
+    layer.events.data = Mock()
+
+    # Required because otherwise the context blocker in the select function in points_mouse_bindings lacks attributes.
+    layer.events.data.blocker().__enter__ = Mock(
```

Comment:
This seems so strange. Have we seen this issue before with event blockers?

Reply:
I did not know of `MagicMock` [fabd929](https://github.com/napari/napari/pull/6611/commits/fabd9294b18ec8a77cfa39cd043af95225647aac). This fixes that part.